src/ACSetInterface.jl

```diff
@@ -4,14 +4,16 @@ export ACSet, acset_schema, acset_name, dom_parts, subpart_type,
   add_part!, add_parts!, set_subpart!, set_subparts!, clear_subpart!,
   rem_part!, rem_parts!, cascading_rem_part!, cascading_rem_parts!, gc!,
   copy_parts!, copy_parts_only!, disjoint_union, tables, pretty_tables,
-  @acset, constructor, PartsType, DenseParts, MarkAsDeleted
+  @acset, constructor, PartsType, DenseParts, MarkAsDeleted, remove_freevars!
```

Comment:
For consistency with `rem_part(s)!`, I think this function should be named `rem_free_vars!`.